static/locales/en-US.yaml

```diff
@@ -530,65 +523,63 @@ Comments:
   Hide: Hide
   Replies: Replies
   Reply: Reply
-  There are no comments available for this video: There are no comments available
-    for this video
-  Load More Comments: Load More Comments
+  There are no comments available for this video: No comments for this video
```

Comment:
If we go with 'available' above, then it would improve consistency if we add it here as well

Reply:
Not sure how it fails though.

Reply:
It simply displays the text to my knowledge, if it does not error out with an error code